README.md

```diff
@@ -52,8 +52,8 @@ java -jar migration-utils-{version}-driver.jar <relative-or-absolute-path-to-con
 | PID              | fedora3model:PID†                   | yul:328697               |
 | state            | fedoraaccess:objState               | Active                   |
 | label            | fedora3model:label†                 | Elvis Presley            |
-| createDate       | premis:hasDateCreatedByApplication  | 2015-03-16T20:11:06.683Z |
-| lastModifiedDate | metadataModification                | 2015-03-16T20:11:06.683Z |
+| createDate       | fcrepo:created                      | 2015-03-16T20:11:06.683Z |
+| lastModifiedDate | fcrepo:lastModified                 | 2015-03-16T20:11:06.683Z |
 | ownerId          | fedora3model:ownerId†               | nruest                   |
```

Comment:
Is `ownerId` the same as `fedora:createdBy`?

Reply:
It's not.  In F3, you can set owner id arbitrarily.  I think it was originally added so that it could be used in XACML policies.  There is information we might later be able to harvest from the audit datastream that indicates which user made the updates.